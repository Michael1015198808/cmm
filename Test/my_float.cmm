int __init__() {
    0.7;
    12.43;
    9.00;
    01.23E12;
    43.e-4;
    .5E03;
    .5E+03;
}
