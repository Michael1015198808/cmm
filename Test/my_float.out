Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: __init__
        LP
        RP
      CompSt (1)
        LC
        StmtList (2)
          Stmt (2)
            Exp (2)
              FLOAT: 0.700000
            SEMI
          StmtList (3)
            Stmt (3)
              Exp (3)
                FLOAT: 12.430000
              SEMI
            StmtList (4)
              Stmt (4)
                Exp (4)
                  FLOAT: 9.000000
                SEMI
              StmtList (5)
                Stmt (5)
                  Exp (5)
                    FLOAT: 1229999964160.000000
                  SEMI
                StmtList (6)
                  Stmt (6)
                    Exp (6)
                      FLOAT: 0.004300
                    SEMI
                  StmtList (7)
                    Stmt (7)
                      Exp (7)
                        FLOAT: 500.000000
                      SEMI
                    StmtList (8)
                      Stmt (8)
                        Exp (8)
                          FLOAT: 500.000000
                        SEMI
        RC
